src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateHelper.java

```diff
@@ -208,11 +208,24 @@ caCert, newSerialNumber(), notBefore, notAfter, subject, publicKey
      * @throws CertificateEncodingException If unable to get certificate encoding
      */
     public static String toPem(X509Certificate certificate) throws IOException, CertificateEncodingException {
-        PemObject pemObject = new PemObject(PEM_BOUNDARY_CERTIFICATE, certificate.getEncoded());
```

Comment:
`toPem(X509Certificate... certificates)` will accept a single certificate. This can be removed